include/difops.hxx

```diff
@@ -193,7 +193,7 @@ const Field3D Grad2_par2(const Field3D &f, CELL_LOC outloc=CELL_DEFAULT);
  * These are a simple way to do staggered differencing
  */
 const Field3D Grad_par_CtoL(const Field3D &var);
-const Field3D Vpar_Grad_par_LCtoC(const Field &v, const Field &f);
+const Field3D Vpar_Grad_par_LCtoC(const Field &v, const Field &f, REGION region=RGN_NOY);
```

Comment:
Is this region consistent with the old behaviour? I couldn't see it in the old implementation but didn't look that closely.